manifests/filter-by-terms/search/queue.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
+metadata:
+  name: search-queue
+  namespace: journal--filter-by-term
+spec:
+  interval: 1m0s
+  sourceRef:
+    kind: GitRepository
+    name: journal-team-deployment
+    namespace: flux-system
+  path: ./kustomizations/search/queue
+  prune: true
+  targetNamespace: journal--filter-by-term
+  postBuild:
+    substitute:
+      # deployment vars
+      env: prod
+      aws_region: "${aws_region}"
+      aws_account_id: "512686554592"
+
+      # Which SNS topics should the queue be subscribed to?
+      bus_articles_topic_arn: arn:aws:sns:us-east-1:512686554592:bus-articles--filter-by-term
```

Comment:
Assuming we remove, rather subscribe to different SNS topics, these values might be better as dummy values, to signify that they we don't expect notifications from these?